### ISSUE TITLE
Desmume boot from firmware setting not working

### ISSUE THREAD

**Issue by KirbyFan102** [TASVideos/desmume#457]
It only displays a white screen. As far as I know, there aren't any issues with my configuration, nor the BIOS images.
![Capturar](https://user-images.githubusercontent.com/30026660/127753285-24db86a5-34ec-4853-bca7-af0e116cf4bf.PNG)

**Comment by zeromus** [comment]
Works for me with all the same options you have checked. Delete the .dfc "user settings from file" and find a firmware that's not broken.

**Comment by KirbyFan102** [comment]
> 
> 
> Works for me with all the same options you have checked. Delete the .dfc "user settings from file" and find a firmware that's not broken.

Now hold on a minute! Where do I find this .dfc file? I don't seem to have it.

**Comment by KirbyFan102** [comment]
> 
> 
> Works for me with all the same options you have checked. Delete the .dfc "user settings from file" and find a firmware that's not broken.

The problem continues to occur. I have deleted the .dfc file and downloaded a new firmware. Nothing is happening.
Please don't close people's issues until you are actually sure they are resolved.

**Comment by zeromus** [comment]
You haven't found firmware that's not broken yet, then. Does your desmume work _without_ the booting from firmware?

**Comment by KirbyFan102** [comment]
It does, but I would still like to get the firmware working. I remember being able to on earlier versions.
But where can I find a non-broken version of this firmware? Also, what is the current Desmume version? I'm worried mine might not be up to date.

**Comment by zeromus** [comment]
1. We can't help you find roms, of course
2. You can download the latest version of desmume from the desmume download page

**Comment by KirbyFan102** [comment]
Okay, I figured out the problem. It wasn't actually because the firmware was broken, or that I was using an outdated build. I had simply misunderstood how the boot from firmware option actually worked. I thought it would show up as soon as I booted the emulator itself, as opposed to when I loaded a ROM. 

I never once tested trying to load a rom after I put the firmware because I hadn't realized that's what I was supposed to be doing. It was probably working fine a long time ago. I'm sorry for bothering you with this.
